src/mixins/filing-mixin.ts

```diff
@@ -1,5 +1,5 @@
 import { Component } from 'vue-property-decorator'
-import { DateMixin } from '@/mixins'
+import DateMixin from './date-mixin'
```

Comment:
Why change this? Not that we would do it, but it makes THIS file less portable (say, to another folder).

In the old code, we could do a global search for "@/mixins" to see where they're used, but now we cannot.

Reply:
Sev, as I mentioned in a comment above, many of the unit tests were failing "TypeError: Class extends value undefined is not a constructor or null"

This is a special case only in `filing-mixin.ts`. As I made some research online, I find out this type of import is what caused the issue.

Reply:
So your change to mixins/index.ts didn't fix it?

Reply:
It did not. These two changes have to go together to fix the unit tests!

Reply:
OK. If it's a problem in the future then we'll fix it then.